src/dlls/mscoree/mscoree.cpp

```diff
@@ -1137,13 +1141,22 @@ STDAPI LoadLibraryShimInternal(LPCWSTR szDllName, LPCWSTR szVersion, LPVOID pvRe
     
```

Comment:
This method is not used in CoreCLR. It would be better to ifdef the whole method with `#ifndef FEATURE_CORECLR` instead of trying to "fix it".